extensions/azurecore/package.json

```diff
@@ -172,6 +172,14 @@
           "dark": "resources/dark/add_to_server_list_inverse.svg",
           "light": "resources/light/add_to_server_list.svg"
         }
+      },
+      {
+        "command": "azure.resource.connectazureDataExplorer",
```

Comment:
`connectAzureDataExplorer`

Reply:
Although - when we can could we try to use Kusto internally? `Azure Data Explorer` is going to get pretty confusing with our own `Data Explorer` concept in ADS.